internal/server/v0/statvarpath/golden/get_statvar_path/school.json

```diff
@@ -1,6 +1,6 @@
 {
   "path": [
     "dc/g/Person_EducationalAttainment",
-    "dc/g/Education"
+    "dc/g/Demographics"
```

Comment:
@pradh is this expected?

Reply:
On autopush it shows up under both Demographics and Education?

Reply:
autopush has some deployment issue so it's stale. For now we should rely on this PR..

Reply:
what cache is this?

Reply:
SVG hierarchy cache in frequent also has Person_EducationalAttainment under both Demographics and Education.

`./check_bt 'd/1' frequent | ./cache_parse > /tmp/svg.txt`

Reply:
I see, in BigQuery it is only specializationOf dc/g/Education

https://screenshot.googleplex.com/3Rgv9Ar6zVvPjxr

Reply:
Should we roll this to staging and validate?

Reply:
sg, I will merge this PR and tested on staging